npm/angular/rollup.config.mjs

```diff
@@ -8,6 +8,7 @@ const config = {
     '@angular/platform-browser-dynamic/testing',
     'zone.js',
     'zone.js/testing',
+    '@angular/core/rxjs-interop',
```

Comment:
@jennifer-shehane i noticed this was incorrectly being bundled into the mount handler. This might be worth adding a change item for, but I also do not want the validation to fail for 2 weeks ☹️